examples/llava/README-minicpmv2.6.md

```diff
@@ -0,0 +1,107 @@
+## MiniCPM-V 2.6
+
+### Prepare models and code
+
+Download [MiniCPM-V-2_6](https://huggingface.co/openbmb/MiniCPM-V-2_6) PyTorch model from huggingface to "MiniCPM-V-2_6" folder.
+
+Clone llama.cpp:
+```bash
+git clone git@github.com:OpenBMB/llama.cpp.git
+cd llama.cpp
+git checkout minicpmv-main
+```
+
+### Usage of MiniCPM-V 2.6
+
+Convert PyTorch model to gguf files (You can also download the converted [gguf](https://huggingface.co/openbmb/MiniCPM-V-2_6-gguf) by us)
```

Comment:
Quick comment on the HF Hub repo: https://huggingface.co/openbmb/MiniCPM-V-2_6-gguf

Adding usage instructions on the repo model card would be good so that people who discover the repo can quickly run it via llama.cpp too.

Reply:
Good idea. I will update it on hf later.